#!/usr/bin/env python
PACKAGE = "templatematching"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
#bool
gen.add("use_cloud_reso",    bool_t,    0, "using cloud resolution",  False)
gen.add("use_hough",    bool_t,    0, "true for hough,false for GC",  False)

#param
gen.add("model_ss",    double_t,    0, "model ss",  0.01,  0.005, 20)
gen.add("scene_ss",    double_t,    0, "scene ss",  0.01,  0.005, 20)
gen.add("rf_rad",    double_t,    0, "rf radius",  0.015,  0.005, 20)
gen.add("descr_rad",    double_t,    0, "describer radius",  0.01,  0.005, 20)
gen.add("cg_size",    double_t,    0, "model ss",  0.01,  0.005, 20)
gen.add("cg_thresh",    double_t,    0, "model ss",  3.0,  1.0, 20.0)


exit(gen.generate(PACKAGE, "templatematching", "matchingparam"))
